engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
@tweise I would prefer to handle deprecation of "dt" prefix as part of a separate PR as the scope of the change is way beyond this PR. In addition, I would focus on the user experience and IMO, it will be quite confusing to have some settings that start with "dt" and others that start with "apex".